src/pybind/mgr/dashboard/frontend/src/app/shared/directives/autofocus.directive.spec.ts

```diff
@@ -18,7 +18,7 @@ export class PasswordFormComponent {}
 @Component({
   template: `
     <form>
-      <input id="x" type="checkbox" autofocus />
+      <input id="x" type="checkbox" [autofocus]="1 > 0" />
```

Comment:
To verify the function use of `autofocus` you should add a new input element which will be focused on some Boolean. Currently the focus will always be true. It would be nice if you would also make sure that it can be false.

My idea was something like this:
```suggestion
      <input id="x" type="checkbox" [autofocus]="!edit" />
      <input id="z" type="checkbox" [autofocus]="() => edit" />
```

Reply:
I assume you need another component mock using autofocus without any conditionals and two additional tests.

Reply:
Done